lib/worker.js

```diff
@@ -82,20 +106,22 @@ class Worker extends BaseService {
 
     _start() {
         // Worker.
-        process.on('SIGTERM', () => this.stop()
-        .then(() => {
-            this._logger.log('info/service-runner/worker', {
-                message: 'worker shutting down',
-                worker_pid: process.pid
+        process.on('SIGTERM', this._shutdownHandler = () => {
```

Comment:
Why introducing a block `{}`?

Reply:
I don't follow. We can discuss tomorrow.

Reply:
I mean, why did you wrap the handler in `{}`? There's no need for that and simple () => this.stop().... looks nicer.

Reply:
Done